### PR TITLE
add site

### PR DESCRIPTION
**Pull request by NINJACLUBREAL**
!!!if I did some thing wrong please fix it and I will deploy with NETLIFY!

## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X ] The file is in the `domains` folder and is in the JSON format.
- [ X] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [X ] The website is reachable.  
- [ ] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [X ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/ninjaclubreal.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "repo": "https://github.com/NINJACLUBREAL/mysite",
+  "owner": {
+    "username": "NINJACLUBREAL",
+    "email": "bee411769@gmail.com"
+  },
+"record": {
+        "A": [
+            "75.2.60.5"
+        ]
+  "URL": "https://ninjaclubdev.is-a.dev"
```
Comment (DEV-DIBSTER) [inline domains/ninjaclubreal.json:12]: Cannot use A records and URL records in conjunction with each other.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/ninjaclubreal.json (hunk 1 above)
Comment (NINJACLUBREAL) [reply domains/ninjaclubreal.json:12]: ok will fix that this is my first time doing this

Comment (NINJACLUBREAL) [reply domains/ninjaclubreal.json]: > Cannot use A records and URL records in conjunction with each other.

i think I fixed it

File: domains/ninjaclubreal.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "IDK",
+  "repo": "https://github.com/NINJACLUBREAL/mysite",
+  "owner": {
+    "username": "NINJACLUBREAL",
+    "email": "bee411769@gmail.com"
+  },
+"record": {
+      
+  "URL": "https://ninjaclubdev.is-a.dev"
```
Comment (DEV-DIBSTER) [inline domains/ninjaclubreal.json:10]: This URL doesn't exist and just redirects back to our root domain.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by orxngc** [comment]
Closing as stale.

File: domains/ninjaclubreal.json (hunk 2 above)
Comment (NINJACLUBREAL) [reply domains/ninjaclubreal.json:10]: change to to https://ninjaclub.rf.gd than

**Pull request closed without merging**